autoload/snipe/core.vim

```diff
@@ -153,6 +153,45 @@ function! s:SafeSetLine(lnum, line) " {{{
   call setline(a:lnum, a:line)
   redraw
 endfunction
+
+let s:buf = 0
```

Comment:
Is there a way to make this not global, and instead pass it around as a parameter?

Reply:
I can't find a way to do that. I make it global to avoid creating a new buffer every time.